pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -16,6 +16,7 @@ import (
 type StorageClusterSpec struct {
 	ManageNodes       bool                          `json:"manageNodes,omitempty"`
 	InstanceType      string                        `json:"instanceType,omitempty"`
+	HostNetwork       bool                          `json:"hostNetwork,omitempty"`
```

Comment:
I think because `bool` defaults to false in golang, this will result in the original behavior if not specified in the crd. This might be worth pointing out.

Reply:
Right. I'll see how I can add some notes about this in the CRD.

If we were targeting K8S-1.16 we could use https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#defaulting to explicitly set a default value. For now, we'll have to live with docs.